tests/basic/storage_pool_test.go

```diff
@@ -5188,3 +5190,179 @@ var _ = Describe("{ResizePoolDrivesInDifferentSize}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{VolDeletePoolExpand}", func() {
+	//1) Deploy px with cloud drive.
+	//2) Create a large volume on that pool and write 200G on the volume.
+	//3) Update the label for the pool before expand
+	//4) perform volume delete
+	//4) Expand by resize the pool when delete is in progress
+	//5) Check the alert for the pool expand
+	//6) check the labels after pool expand
+	var testrailID = 51285
+	//testrailID corresponds to : https://portworx.testrail.net/index.php?/tests/view/51285
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("VolDeletePoolExpand", "Delete volume which has ~200G data and do an expansion of pool by resize", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	var newContexts []*scheduler.Context
+	stepLog := "should get the existing storage node and write ~200G data to a volume"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("voldeletepoolexpand-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		log.Infof("Need to check if volume is close to 200G occupied")
+		vol, err := getVolumeWithMinimumSize(contexts, 90)
```

Comment:
should it be 190 ? vol, err := getVolumeWithMinimumSize(contexts, **190**)

Reply:
Current vdbench is writing only upto 98G. Once we can comeup with another spec, I will change this soon